packages/apps/esm-login-app/src/config-schema.ts

```diff
@@ -36,6 +36,12 @@ export const configSchema = {
       _default: 8,
       _description: "The number of locations displayed on location picker",
     },
+    resultsInOneCycle: {
```

Comment:
This is going to be a hard one to name properly... I'm not even certain if it is something we need to be user-configurable, but we can probably come up with a better name for this option.

```suggestion
    locationsPerRequest: {
```